build_projects/dotnet-cli-build/DebTargets.cs

```diff
@@ -13,14 +13,14 @@ namespace Microsoft.DotNet.Cli.Build
     public class DebTargets
     {
         [Target(nameof(GenerateSdkDeb))]
-        [BuildPlatforms(BuildPlatform.Ubuntu)]
+        [BuildPlatforms(BuildPlatform.Ubuntu, "14.04")]
```

Comment:
My understanding was in infrastructure did not support multiple versions of Ubuntu yet.